aiproteomics/modelgen/save_model.py

```diff
@@ -0,0 +1,78 @@
+import os
+import tf2onnx
+
+def save_model(
+        model, 
+        name,
+        framework = 'keras',
+        output_format = 'onnx',
+        output_dir = './aiproteomics/modelgen/saved_models/',
+        overwrite = True,
+    ):
+
+    """Function to save the model.
+
+    Args:
+        model (str): the model to be saved
+        
+        name (str): the name to save the model as
+
+        framework (str): the framework used to generated the model.
+            Currently only keras is implemented.
+
+        output_format (list or str): format or list of formats to save the model as.
+            Currently only 'onnx' and 'keras' are implemented.
+            Defaults to 'onnx'.
+
+        output_dir (str): directory to save the model
+
+        overwrite (bool): 
+            if True: any existing file model with the same name will be overwritten.
+            if False: will append a number behind the file name to create a new file
+    """
+
+    # Check save formats    
+    valid_output_formats = ['onnx', 'keras']
```

Comment:
I would move this to the top of the file (under the imports) and call it `VALID_OUTPUT_FORMATS`. Eventually we will put it somewhere central (maybe `definitions.py` or something with that kind of name). For now, it's useful to see top level config stuff at the top so we can collect it later.

Reply:
We'll probably make this enums in the end.